Add consideration for: `flask` (#22)
Sigmapitech/area-techno-tester: considerations/back-flask/README.md

# Consideration for Flask

## Overview
[Flask](https://flask.palletsprojects.com/) is a lightweight and flexible Python web framework based on WSGI (Web Server Gateway Interface). It follows a “micro” philosophy, providing only the essentials to get an application running, while leaving developers free to choose additional tools and libraries. Flask is known for its simplicity, readability, and large ecosystem of extensions, making it one of the most popular Python web frameworks.

## Why we considered it?
We considered Flask because it provides a minimal, easy-to-use framework for building web applications and APIs.
Our project requires a backend service, and Flask’s simplicity allows us to quickly prototype and deliver endpoints without the complexity of larger frameworks. It integrates well with databases, authentication libraries, and templating engines, giving us freedom to build only what we need.
Flask also has a long track record, which means strong community support and a wealth of resources for troubleshooting.

## Who in the group has prior knowledge about this tech?

> None of the group members have prior experience in Flask, which means there may be a learning curve involved in backend development. However, Flask's simplicity and extensive documentation should help us get up to speed quickly.

Author: ```suggestion
Yohann has prior experience with Flask, and Flask's simplicity extensive documentation could help get up to speed quickly.
```
